VideoLocker/src/main/java/org/edx/mobile/player/PlayerController.java

```diff
@@ -763,7 +766,9 @@ public void onClick(View v) {
 //          }
         }
     };
-
+    /**
+     * Currently unused 3/4/2015

```

Comment:
Well I just mean if the code isn't being used we should remove it? Or are you saying, we might need to hook it up?